### ISSUE TITLE
Non-Latin characters on linux dont work on some distros

### ISSUE THREAD

**Issue by Damianu** [Facepunch/garrysmod-issues#3154]
Providing extended = true to font table doesnt do anything for me(Arch Linux). This problem doesnt occur in csgo or steam itself as I got fallbacks so I dont think it's source's fault.
Russian characters work for example(no matter if extended or not), but japanese wont.

**Comment by mcd1992** [comment]
Try installing the ttf noto fonts.

**Comment by Damianu** [comment]
noto-fonts-20170307-2 is up to date. However I do not fallback to them, may try that.

**Comment by Damianu** [comment]
It doesn't help, I am not missing fonts, just gmod doesnt want to render them for some reason :/

**Comment by mcd1992** [comment]
Post some strings to test with and screenshots if you can.

**Comment by Damianu** [comment]
Basically all non-latin characters will render as box. 
For example my name: 
D.ツ
Is just D.[box]. It works in steam overlay/steam/csgo. Worth to mention is that I replaced verdana with liberation sans and forced fallbacks to WenQuanYiZenHei, without that it was even more annoying as those characters werent rendered at all. I can post screenshot in few minutes.

**Comment by Damianu** [comment]
Found workaround, for some reason my chat was using "TeX GyreHeros" (It should use Verdana  I think? I even got it installed), so I just forced it to use font with non-latin characters and then normal font as fallback, you can do that by:
```
 <match target="pattern">
   <test qual="any" name="family"><string>TeX Gyre Heros</string></test>
   <edit name="family" mode="assign" binding="same"><string>WenQuanYiZenHei</string></edit>
 </match>
```
In fontconfig.

**Comment by Damianu** [comment]
It behaves weird, seems like font has to have all characters because fallback doesnt work, however FontAwesome doesnt work at all, it always displays those rects no matter if extended is enabled or not, on windows it works perfectly fine and you can get nice glyphs :(.

**Comment by willox** [comment]
There's 0 support for the extended option in Linux builds of the game. Hopefully that'll change in the future, but it's been like that for a while now.

**Comment by Damianu** [comment]
Yeah I hope it will, cuz currently a lot of nicknames are unreadable and also some addons use extended fonts for icons, those are kinda unusable on linux. But thanks for information.
